app/controllers/application_controller.rb

```diff
@@ -4,6 +4,18 @@ class ApplicationController < ActionController::Base
 
   private
 
+  def load_commentable
+    request_path = request.path.split('/')
+    if request_path[1] == ('en' || 'uk')
```

Comment:
instead of doing such strange thing you could look into params, they always have the controller and action there.

Reply:
@omaik then I wont be able to get right commentable resource from comments params